.github/workflows/build.yaml

```diff
@@ -37,10 +39,11 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-go@v3
+      - uses: actions/checkout@v4
+      - name: Set up Go
+        uses: actions/setup-go@v5
         with:
-          go-version: '1.23'
+          go-version: 'stable'
```

Comment:
I think it's better to set the go-version from go mod file to avoid mismatch in versions between CI/CD and codebase